content/en/docs/concepts/cluster-administration/cloud-providers.md

```diff
@@ -134,15 +125,6 @@ If you wish to use the external cloud provider, its repository is [kubernetes/cl
 The GCE cloud provider uses the hostname of the node (as determined by the kubelet or overridden with `--hostname-override`) as the name of the Kubernetes Node object.
 Note that the first segment of the Kubernetes Node name must match the GCE instance name (e.g. a Node named `kubernetes-node-2.c.my-proj.internal` must correspond to an instance named `kubernetes-node-2`).
 
-## HUAWEI CLOUD
-
-If you wish to use the external cloud provider, its repository is [kubernetes-sigs/cloud-provider-huaweicloud](https://github.com/kubernetes-sigs/cloud-provider-huaweicloud).
```

Comment:
L137-139 Content in project; keep as is

Reply:
If L137-139 are kept, line 141-145 should be kept as well?